eventmesh-admin/src/main/java/org/apache/eventmesh/admin/enums/Errors.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.admin.enums;
+
+import static org.apache.eventmesh.admin.common.ConfigConst.COLON;
+
+import org.springframework.http.HttpStatus;
+
+import lombok.Getter;
+
+/**
+ * An enumeration class that conforms to the RESTful specifications and custom error reporting requirements.
+ * <ul>
+ *   <li>The 'code' field is used to return the HTTP status code using {@link HttpStatus}.</li>
+ *   <li>The 'type' field represents the major category of the error.</li>
+ *   <li>the 'desc' field represents the detailed subcategory and information of the error.</li>
+ * </ul>
+ */
+
+@Getter
+public enum Errors {
+
+    SUCCESS(HttpStatus.OK, Types.SUCCESS, "Operation success."),
```

Comment:
The naming of the `Errors` class is appropriate for its usage because it needs to be propagated within the `BaseException`, so this naming is the most suitable.

The `SUCCESS` enumeration is always statically referenced, so there won't be a usage like `Errors.SUCCESS`. The usage of the `SUCCESS` enumeration is consistent with other XXX_ERRs, so there's no need to create a separate file for it, and I think this approach is appropriate.

---

Errors类的命名符合它的用法，因为它需要在BaseException中被传播，因此这个命名是最合适的。

SUCCESS枚举总是静态引用的，因此不会出现Errors.SUCCESS的用法。SUCCESS枚举的用法和其它XXX_ERR是一致的，没有必要为它单独新建一个文件，因此我觉得这样是合适的。

Reply:
Yes, I will rename it to 'Status,' which was its original name at first.

---

是的。我会将它改名为`Status`，这也是它一开始的命名。